src/components/progress/Progress.vue

```diff
@@ -0,0 +1,108 @@
+<template>
+    <div class="progress-wrapper">
+        <progress
+            ref="progress"
+            class="progress"
+            :class="newType"
+            :max="max">{{ newValue }}</progress>
+        <p
+            v-if="showValue"
+            class="progress-value has-text-white"><slot>{{ newValue }}</slot></p>
+    </div>
+</template>
+
+<script>
+
+export default {
+    name: 'BProgress',
+    props: {
+        type: {
+            type: String | Object,
+            default: 'is-darkgrey'
+        },
+        size: String,
+        value: {
+            type: undefined | Number,
```

Comment:
There is no need to specify undefined type

Reply:
My mistake! i'll fix it!